src/workflow-engine/charms/temporal-worker/oci_factory/activities/find_images_to_update.py

```diff
@@ -122,12 +121,17 @@ def find_released_revisions(releases_json: dict) -> dict:
         uber_img_trigger = {"version": 1, "upload": []}
         # We'll also need to find which tags (channels) to release the new
         # rebuilds to
-        docker_tags_url = f"https://hub.docker.com/v2/repositories/%%%/{image}/tags"
+        # TODO: Get rid of this once we have a proper DB where to store all the
+        # image information.
+        # This is a bit nasty as these APIs return paginated results
+        # and don't offer enough querying parameters to filter the results.
+        ecr_tags_url = "https://api.us-east-1.gallery.ecr.aws/describeImageTags"
```

Comment:
why you don't use boto3?

Reply:
why the extra requirement?

Reply:
I'll make the changes in a new PR if you think there's value in using boto3. For now, I need to test this so I'll go ahead and merge.